_includes/regions/sponsor_silver.html

```diff
@@ -1,5 +1,5 @@
 <div class="sponsor silver-sponsor clearfix">
-  <h3>Silver sponsor</h3>
+  <div class="col-md-12"><h3>Silver sponsors</h3></div>
```

Comment:
Is this applied for the styling only? Maybe we can style the `h3` tag and not use the grid classes in this section.
I think the indention is off also also in this file.

Reply:
I had to do this because otherwise the h3 is acting as a float left. I will see what I can do.